scripts/conventional-changelog/index.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+var parserOpts = {
```

Comment:
I never had to pass a script to conventional-changelog, what are the benefits here from the regular conventional-changelog usage?